crates/wasi-common/build.rs

```diff
@@ -133,7 +133,7 @@ mod wasm_tests {
             &stemstr.replace("-", "_")
         )?;
         writeln!(out, "        setup_log();")?;
-        write!(
+        writeln!(
```

Comment:
How come this one sneaked in into this PR? :-)

Reply:
Just something I noticed when I looked at the generated test code and saw the misaligned code.